src/mmw/mmw/settings/base.py

```diff
@@ -130,6 +130,7 @@ def get_env_setting(setting):
 CELERY_CREATE_MISSING_QUEUES = True
 CELERY_CHORD_PROPAGATES = True
 CELERY_CHORD_UNLOCK_MAX_RETRIES = 60
+# CELERYD_CONCURRENCY = 2
```

Comment:
Good point about Spark. I definitely don't advocate tweaking it until we have some kind of test in place that can mimic each type of potential resource or concurrency problem, but given the time/money required it may prove to be a worthy trade at some point.